java/Oleksii-Riabov/src/com/zoolatech/lecture4/tasks/_7/Task7.java

```diff
@@ -0,0 +1,45 @@
+package com.zoolatech.lecture4.tasks._7;
+
+import java.util.Arrays;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+/**
+ * Create a method that accepts a string of digits and after each digit
+ * appends its string representation in parentheses.
+ */
+
+public class Task7 {
+    public static void main(String[] args) {
+        String string = "12345678910";
+        System.out.println(stringRepresentation(string));
+    }
+
+    public static String stringRepresentation(String string) {
+        if (string.matches("[0-9]+") && string.length() > 2) {
+            return Stream.of(string)
+                    .map(s -> s.split("")).
+                    flatMap(s -> Arrays.stream(s).sequential())
+                    .map(s -> check(s))
+                    .collect(Collectors.joining());
+        } else {
+            throw new IllegalArgumentException("Pls, enter numbers only!");
+        }
+    }
+
+    public static String check(String string) {
```

Comment:
What do we check here? I think this method should be renamed

Reply:
Already rename this method.